src/main/kotlin/racingcar/ui/RaceResultStorage.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.ui
+
+import racingcar.domain.Cars
+import racingcar.domain.RaceResult
+
+class RaceResultStorage(
+    results: List<CarMove> = emptyList()
+) : RaceResult {
+
+    private val _results: MutableList<CarMove> = results.toMutableList()
+
+    override fun save(cars: Cars) {
+        val carMoves = cars.getPositions().map {
+            CarMove(it.position)
+        }
+        ResultView.printRaceResult(carMoves)
+        _results.addAll(carMoves)
+        Thread.sleep(500) // 자동차 경주 사이의 결과를 0.5초씩 딜레이시켜 실시간으로 출력되는 것처럼 보이도록 만들기
+    }
+
+    val results
+        get() = _results.toList()
```

Comment:
선언순서를 아래와 같이 정의하는것이 어떨까요 ? 
https://kotlinlang.org/docs/coding-conventions.html#class-layout
```suggestion
    private val _results: MutableList<CarMove> = results.toMutableList()
    val results
        get() = _results.toList()
        
    override fun save(cars: Cars) {
        val carMoves = cars.getPositions().map {
            CarMove(it.position)
        }
        ResultView.printRaceResult(carMoves)
        _results.addAll(carMoves)
        Thread.sleep(500) // 자동차 경주 사이의 결과를 0.5초씩 딜레이시켜 실시간으로 출력되는 것처럼 보이도록 만들기
    }    
```

Reply:
자바에서는 getter가 최하위였던 걸로 기억하는데, 코틀린은 프로퍼티로 묶여 있다 보니까 선언 순서가 다르군요... 😎 감사합니다!!